README.md

```diff
@@ -19,11 +19,13 @@ There are more details about our approach [in this blog post](https://4ta.uk/p/s
 
 `ghcide` already exports the following features via the lsp protocol:
 
-* Display error messages (parse errors, typecheck errors, etc.) and enabled warnings.
-* Offer quickfix for unused imports.
-* Go to definition in local package.
-* Display definitions.
-* Organize imports.
+| Feature | LSP name |
+| - | - |
+| Display error messages (parse errors, typecheck errors, etc.) and enabled warnings. | diagnostics |
+| Go to definition in local package | definition  | 
+| Display type and source module of values | hover |
+| Remove redundant imports, replace suggested typos for values and module imports, fill type holes, add suggested ghc extensions,  | codeAction (qickfix) |
```

Comment:
Typo! Maybe some spelling quickfix action can help ...

Reply:
Wow, I had to stare at this for 60s to see the mistake …